modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc

```diff
@@ -205,7 +205,7 @@ For high availability, Redpanda Cloud uses a control plane and data plane archit
 
 Clusters are configured and maintained in the control plane, but they remain available even if the network connection to the control plane is lost.
 
-NOTE: In the user interface, when you're at the organization (org) level or the namespace level, but you haven't yet selected a cluster, you're in the control plane. This is where you can select, create, and delete clusters, namespaces, and networks. When you're at the cluster level working with topics, consumer groups, and connectors, you're in the data plane.
+NOTE: In the user interface, when you're at the organization (org) level or the resource group level, but you haven't yet selected a cluster, you're in the control plane. This is where you can select, create, and delete clusters, resource groups, and networks. When you're at the cluster level working with topics, consumer groups, and connectors, you're in the data plane.
```

Comment:
I think a diagram that shows what is in the CP vs DP would be helpful--not the one below, which does not show all of the specific elements encompassed in each.

Reply:
@Feediver1: I put this in a [new issue](https://github.com/redpanda-data/documentation-private/issues/2411), since the diagram will take some time and CD/DP isn't related to this issue.